cloud/amazon/ec2_metric_alarm.py

```diff
@@ -184,8 +184,11 @@ def create_metric_alarm(connection, module):
         comparisons = {'<=' : 'LessThanOrEqualToThreshold', '<' : 'LessThanThreshold', '>=' : 'GreaterThanOrEqualToThreshold', '>' : 'GreaterThanThreshold'}
         alarm.comparison = comparisons[comparison]
 
-        dim1 = module.params.get('dimensions', {})
+        dim1 = module.params.get('dimensions')
```

Comment:
You say "Adding {} to module.params.get does not work". Is this a problem with a specific python version?

```
>>> a = dict(b='c', d='e')
>>> a.get('b', {})
'c'
>>> a.get('f', {})
{}
```
In what way does it not work?